hack/generate_tests.py

```diff
@@ -58,7 +58,7 @@ def run_kustomize_build(repo_root, package_dir):
   logging.info("Creating directory %s", output_dir)
   os.makedirs(output_dir)
 
-  subprocess.check_call([os.environ.get("KUSTOMIZE_BIN", "kustomize"), "build", "--load_restrictor", "none",
+  subprocess.check_call([os.environ.get("KUSTOMIZE_BIN", "kustomize"), "build",
```

Comment:
Can we keep this until we have found a tested replacement to use instead of load_restrictor none?

Reply:
Just pushed to commit to reverse this change